modules/alloy/src/main/ts/ephox/alloy/ui/single/TieredMenuSpec.ts

```diff
@@ -41,7 +41,6 @@ const make: SingleSketchFactory<TieredMenuDetail, TieredMenuSpec> = (detail, _ra
   const buildMenus = (container: AlloyComponent, primaryName: string, menus: Record<string, PartialMenuSpec>): Record<string, MenuPreparation> => Obj.map(menus, (spec, name) => {
 
     const makeSketch = () => Menu.sketch({
-      dom: spec.dom,
```

Comment:
Tried to track this through but not sure I understood. Are we deleting it because it's just coming from `spec` and that'd get folded in with `...spec` anyway? If I did get that right, then shouldn't we delete `items: spec.items` too?

Reply:
That sounds right to me, @metricjs. 
@lnewson ?

Also - is this bit sufficiently covered in tests?

Reply:
Yes, TS 3.9 flagged this line as doing nothing since it was just overwritten by the following line. I check manually and it was always set. You're right that `items: spec.items` can also be removed.

As for tests, I'd say yes as there's a fair few menu based tests that run through this code. However, that does bring up that I should add a test for the `ActiveDisable` issue, as we obviously are missing a test for that.

Reply:
Fixed